README.md

```diff
@@ -11,135 +23,159 @@ Clone the depot_tools repository:
 $ git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
 ```
 
-Add depot_tools to the end of your PATH (you will probably want to put this in your ~/.bashrc or ~/.zshrc). Assuming you cloned depot_tools to /path/to/depot_tools:
+Add depot_tools to the end of your PATH (you will probably want to put this in your ~/.bashrc or ~/.zshrc). Assuming you cloned depot_tools to ./depot_tools:
 
 ```sh
-$ export PATH="$PATH:/path/to/depot_tools"
+$ export PATH="$PATH:$PWD/depot_tools"
 ```
 
 
-### Get the code
+### 2. Get the code
 
 
 Create a chromium directory for the checkout and change to it (you can call this whatever you like and put it wherever you like, as long as the full path has no spaces):
-
 ```sh
-$ mkdir path/to/castanets && cd path/to/castanets
+$ mkdir $PWD/castanets && cd $PWD/castanets
 ```
 
 Download the code using the command below.
-
 ```sh
-$ git clone -b castanets_69 https://github.com/Samsung/castanets src
+$ git clone https://github.com/Samsung/castanets src
 ```
 
 If you did not specify the 'src' directory name at the end of the command, the source code would have been downloaded to the 'castanets' directory. In this case, change the directory name.
-
 ```sh
 $ mv castanets src
 ```
+Change directory to src 
+```
+cd ./src
+```
 
+### 3. How To Build Castanets for Ubuntu (Both a Browser Process and a Renderer Process)
 Install additional build dependencies
-
 ```sh
 $ build/install-build-deps.sh
 ```
 
-
-### Run the sync
-
+#### 3.1 Run the sync
 We need a gclient configuration. To create a .gclient file, run:
 
 ```sh
 $ build/create_gclient.sh
 ```
 
 Once you've run install-build-deps at least once, you can now run the Chromium-specific sync, which will download additional binaries and other things you might need:
-
 ```sh
 $ gclient sync --with_branch_head
 ```
 
 
-### Setting up the build
+#### 3.2 Argument Setup for Build
 
+Chromium uses Ninja as its main build tool along with a tool called GN to generate .ninja files. You can create any number of build directories with different configurations. 
+To create a build directory, run:
+```sh
+$ gn gen --args='enable_castanets=true enable_nacl=false is_debug=false' out/Default
+```
+for faster build, you may add following options
+```
+remove_webcore_debug_symbols=true
+is_component_build=true
+use_jumbo_build=true
+```
 
-Chromium uses Ninja as its main build tool along with a tool called GN to generate .ninja files. You can create any number of build directories with different configurations. To create a build directory, run:
-
+#### 3.3 Build castanets for Ubuntu
+Build castanets (the “chrome” target) with Ninja using the command:
 ```sh
-$ gn gen out/Default
+$ ninja -C out/Default chrome
```

Comment:
How about use autoninja instead of ninja?
It will provides optimal values automatically.

- https://chromium.googlesource.com/chromium/src/+/master/docs/android_build_instructions.md
- https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/refs/heads/master/autoninja.py

Reply:
Changed to autoninja. Thank you